### ISSUE TITLE
'Tensor' object has no attribute 'norms'

### ISSUE THREAD

**Issue by faithyrenne** [bethgelab/foolbox#629]
for eps, advs_ in zip(epsilons, clipped_advs):
        acc2 = accuracy(fmodel, advs_, labels)
        print(f"  Linf norm ≤ {eps:<6}: {acc2 * 100:4.1f} %")
        print("    perturbation sizes:")
        perturbation_sizes = (advs_ - images).norms.linf(axis=(1, 2, 3)).numpy()
        print("    ", str(perturbation_sizes).replace("\n", "\n" + "    "))
        if acc2 == 0:
            break

IN resent training with single attack (LinfPGD)

for calculating the size of perturbations:
'Tensor' object has no attribute 'norms'

**Comment by jonasrauber** [comment]
I guess you copied the `perturbation_sizes = (advs_ - images).norms.linf(axis=(1, 2, 3)).numpy()` line from some example that used EagerPy tensors (for `advs_` and `images`) and you use raw PyTorch tensors?

Either wrap your tensors as EagerPy tensors (like the example code you copied this from) or use PyTorch's own `norm` function (`torch.norm` if I am not mistaken).
